nb5-proof/src/test/java/io/nosqlbench/nb5/proof/WorkloadContainerVerifications.java

```diff
@@ -0,0 +1,213 @@
+            if(shortName.equals("cql-iot-dse.yaml"))
+                continue;
+            //STEP0:Start the test container and expose the 9042 port on the local host.
+            //So that the docker bridge controller exposes the port to our process invoker that will run nb5
+            //and target cassandra on that docker container
+            cass.start();
+            //the default datacenter name
+            String datacenter = cass.getLocalDatacenter();
+            //When running with a local Docker daemon, exposed ports will usually be reachable on localhost.
+            // However, in some CI environments they may instead be reachable on a different host.
+            //the port mapped to the original exposed port of the cassandra image
+            Integer mappedPort9042 = cass.getMappedPort(9042);
+            //the host ip of the cassandra image in the container
+            String hostIP = cass.getHost();
+
+
+           //STEP1: Run the example cassandra workload using the schema tag to create the Cass Baselines keyspace
+            String[] args = new String[]{
+                "java", "-jar", JARNAME, shortName, BASIC_CHECK_IDENTIFIER, "host="+ hostIP, "localdc="+ datacenter, "port="+ mappedPort9042.toString(), "table=keyvalue", "keyspace=baselines"
+            };
+            logger.info("The final command line: " + String.join(" ", args));
+            ProcessResult runSchemaResult = invoker.run("run-workload", 30, args);
+
+
+            //STEP 2 Check runSchemaOut for errors
+            logger.info("Checking if the NB5 command resulted in any errors...");
+            assertThat(runSchemaResult.exception).isNull();
+            String runSchemaOut = String.join("\n", runSchemaResult.getStdoutData());
+            assertThat(runSchemaOut.toLowerCase()).doesNotContain("error");
+            logger.info("NB5 command completed with no errors");
+
+
+            //STEP 4 Check the cluster for created data
+            try (CqlSession session = CqlSession.builder().addContactPoint(new InetSocketAddress(hostIP, mappedPort9042)).withLocalDatacenter(datacenter).build()) {
+                //->Check for the creation of the keyspace baselines
+                logger.info("Checking for the creation of the keyspace \"baselines\"...");
+                ResultSet result = session.execute("SELECT keyspace_name FROM system_schema.keyspaces");
+                List<Row> rows = result.all();
+                boolean keyspaceFound = false;
+                for (Row row : rows) {
+                    if (row.getString("keyspace_name").equals("baselines")) {
+                        keyspaceFound = true;
+                        break;
+                    }
+                }
+                assertTrue(keyspaceFound);
+                logger.info("Keyspace \"baselines\" was found, nb5 command had created it successfully");
+
+                //->Check for the creation of the baselines keyvalue table
```

Comment:
This only matters for specific tests. We should generalize it or disable it until we have a more cohesive approach for the workload variety.